cookbook/entities/managing_tasks.rst

```diff
@@ -0,0 +1,254 @@
+
+    {# extend the base template from the OroUIBundle #}
+    {% extends 'OroUIBundle:actions:update.html.twig' %}
+
+    {# reuse the form theme provided with the OroPlatform #}
+    {% form_theme form with 'OroFormBundle:Form:fields.html.twig' %}
+
+    {# make the current task accessible with the task variable #}
+    {% set task = form.vars.value %}
+
+    {# choose the appropriate action depending on whether a task is created or modified #}
+    {# this variable needs to be named formAction as this is what the base template expects #}
+    {% if task.id %}
+        {% set formAction = path('app_task_update', { 'id': task.id }) %}
+    {% else %}
+        {% set formAction = path('app_task_create') %}
+    {% endif %}
+
+    {% block navButtons %}
+        {# the cancelButton() macro creates a button that discards the
+           entered data and leads the user to the linked controller #}
+        {{ UI.cancelButton(path('app_task_index')) }}
+
+        {# the dropdownSaveButton() macro offers a way to let the user select
+           between different options when saving an entity, the selected option
+           will be passed to the controller handling the request as an additonal
+           parameter #}
+        {{ UI.dropdownSaveButton({
+            'html': UI.saveAndCloseButton() ~ UI.saveAndStayButton()
+        }) }}
+    {% endblock navButtons %}
+
+    {% block pageHeader %}
+        {% if task.id %}
+            {% set breadcrumbs = {
+                'entity': task,
+                'indexPath': path('app_task_index'),
+                'indexLabel': 'Tasks',
+                'entityTitle': task.subject
+            } %}
+            {{ parent() }}
+        {% else %}
+            {% set title = 'oro.ui.create_entity'|trans({ '%entityName%': 'Task' }) %}
+            {{ include('OroUIBundle::page_title_block.html.twig', { title: title }) %}
+        {% endif %}
+    {% endblock pageHeader %}
+
+    {% block content_data %}
+        {% set id = 'task-edit' %}
+        {% set dataBlocks = [{
+                'title': 'General'|trans,
+                'class': 'active',
+                'subblocks': [{
+                    'title': '',
+                    'data': [
+                        form_row(form.subject),
+                        form_row(form.description),
+                        form_row(form.dueDate),
+                        form_row(form.priority),
+                    ]
+                }]
+            }]
+        %}
+
+        {# the data variable is a special variable that is used in the
+           parent content_data block to render the visual content "blocks"
+           of a page #}
+        {% set data = {
+            'formErrors': form_errors(form) ? form_errors(form) : null,
+            'dataBlocks': dataBlocks,
+        } %}
+
+        {{ parent() }}
+    {% endblock content_data %}
+
+.. _controller-entity-grid-create-edit-link:
+
+Linking the Data Grid
```

Comment:
I think you should change this title because create link dies not goes to the grid.